pkgs/tools/networking/cantoolz/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, fetchpatch
+, pytestCheckHook
+, flask
+, pyserial
+, mido
+, numpy
+, bitstring
+}:
+
+buildPythonApplication rec {
+  pname = "cantoolz";
+  version = "3.7.0";
+
+  src = fetchFromGitHub {
+    owner = "CANToolz";
+    repo = "CANToolz";
+    rev = "v${version}";
+    sha256 = "0xkj7zyx6pz866q61c84mdagpgdyd633v85hk7qxhamca33rc4yi";
+  };
+
+  patches = [
+    (fetchpatch {
+      # https://github.com/CANToolz/CANToolz/pull/30
+      url = "https://github.com/CANToolz/CANToolz/pull/30.patch";
```

Comment:
Please fetch a commit and add a comment why this is needed.

Reply:
Updated

Thanks for the feedback.